### PR TITLE
[IOCOM-702] Add remote content api

### PR DESCRIPTION
**Pull request by Garma00**
#### List of Changes

Add a new consumed api for remote content

#### How Has This Been Tested?

no test are needed

#### Types of changes

- [x] Chore (nothing changes by a user perspective)
- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

#### Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: openapi/consumed/api_remote_content.yaml [hunk 1]
```diff
@@ -0,0 +1,524 @@
+openapi: 3.0.1
+info:
+  title: Remote Content - Server implementation
+  version: 9.7.0
```
Comment (michaeldisaro) [inline openapi/consumed/api_remote_content.yaml:4]: maybe this is not really 9.7.0... we are at 1.0.0 I think... for this remote content apis.

File: openapi/consumed/api_remote_content.yaml [hunk 2]
```diff
@@ -0,0 +1,524 @@
...
+      properties:
+        attachments:
+          type: array
+          items:
+            $ref: '#/components/schemas/RemoteContentAttachment'
+        details:
+          type: object
+          properties:
+            markdown:
+              $ref: "#/components/schemas/Markdown"
```
Comment (michaeldisaro) [inline openapi/consumed/api_remote_content.yaml:356]: I'd rather have extracted the whole "details" as a component, than the single Markdown and Subject.

Comment (Garma00) [reply openapi/consumed/api_remote_content.yaml]: [9ad5ee0](https://github.com/pagopa/io-backend/pull/1077/commits/9ad5ee01580411047d44ffec0b64d41fe6a84284)

**Review by acuiuli — APPROVED** [review]

**Review by michaeldisaro — APPROVED** [review]

File: package.json [hunk 3]
```diff
@@ -55,6 +55,7 @@
     "generate:proxy:mitvoucher": "rimraf generated/mitvoucher && gen-api-models --api-spec api_mit_voucher.yaml --out-dir generated/mitvoucher",
     "generate:proxy:zendesk-models": "rimraf generated/zendesk && gen-api-models --api-spec api_zendesk.yaml --out-dir generated/zendesk",
     "generate:proxy:third-party-service": "rimraf generated/third-party-service && gen-api-models --api-spec openapi/consumed/api-third-party.yaml --out-dir generated/third-party-service --request-types --response-decoders --client",
+    "generate:proxy:remote_content_service": "rimraf generated/remote_content_service && gen-api-models --api-spec openapi/consumed/api_remote_content.yaml --out-dir generated/remote_content_service --request-types --response-decoders --client",
```
Comment (michaeldisaro) [inline package.json:58]: maybe better `remote-content-service` like others

**Review by michaeldisaro — CHANGES_REQUESTED** [review]

File: package.json (hunk 3 above)
Comment (Garma00) [reply package.json]: Fixed in [2ab2bea](https://github.com/pagopa/io-backend/pull/1077/commits/2ab2beab5d43f29d5394b91bc72de2812f8804c1)

**Review by michaeldisaro — APPROVED** [review]

**Pull request merged**